src/libs/fsdrv/lv_fs_fatfs.c

```diff
@@ -20,7 +20,7 @@
     #define DIR FF_DIR  /* ESP IDF typedefs `DIR` as `FF_DIR` in its version of ff.h. Use `FF_DIR` in LVGL too */
 #endif
 
-#if LV_FS_FATFS_LETTER == '\0'
+#if (LV_FS_FATFS_LETTER < 'A') || (LV_FS_FATFS_LETTER > 'Z')
```

Comment:
Why do we need to add this restriction?

Reply:
`lv_fs` also needs to support `'/'` as the drive letter, for example: `lv_fs_open(&fa, "/test_files/readtest.txt", LV_FS_MODE_RD);`

Reply:
I wanted to make the code consistent with the error message and the documentation (which states the `[A-Z]:` format).

Using `/` as drive letter should not be encouraged as it can be confused with an absolute path. I also feel like pitfalls will occur if we want to make it work with `LV_FS_DEFAULT_DRIVE_LETTER`.

And I think the intention behind default drive letter was to elegantly handle situations like this, but correct me @kisvegabor

Reply:
This is done to be compatible with absolute paths. It has been used on our platform for quite a long time, and I believe there is no problem with the current design.

Reply:
See this change: https://github.com/lvgl/lvgl/commit/a7084509b51e8c39d4018144b38b5b6a4b5242f8

Reply:
The goal of "default driver letter" feature is to better support absolute and "desktop" paths.

I'm assuming a driver is registered for letter `/` on your platform at the moment. The same driver has to be registered for - let's say - `A`, then `LV_FS_DEFAULT_DRIVE_LETTER` set to be `A`, then all the paths in your code should work the same way as before.

This allows replacing the trick / workaround of using `/` as a drive letter with a more clear and verbose solution.

Reply:
If I understand correct this is what happens if you are using `/` as driver letter:
- `/path/to/files`: is a relative path
- `//path/to/files`: is an absolute path

If this is the case then it seems a little but hacky. Using `/` as driver letter might work, but according to the docs it should have been an upper case letter. How complicated would it be for you to migrate to this new approac?

Reply:
>* `//path/to/files`: is an absolute path

Aren't all LV FS paths absolute relative to the driver mount point?

To me it just looks like the `:` is optional for any drive letter or `/` and `/` does not get any special treatment.

IMO it's okay to lock down the drive letters to `[A-Z]` especially now that a default drive letter is being added.

Reply:
In case it is helpful, the last 2 major projects I undertook both have paths that look like this (absolute path).  This is the "pathing" syntax that comes with the USB MSD library provided by Microchip:

/mnt/mydrive/serial_no/boot/filename.ext

Kind regards,
Vic